R/create-docs.sh

```diff
@@ -43,4 +45,7 @@ Rscript -e 'libDir <- "../../lib"; library(SparkR, lib.loc=libDir); library(knit
 
 popd
 
+# render creates SparkR vignettes
+Rscript -e 'library(rmarkdown); Sys.setenv(SPARK_HOME=tools::file_path_as_absolute("..")); render("pkg/vignettes/sparkr-vignettes.Rmd")'
```

Comment:
you could also set libPaths https://stat.ethz.ch/R-manual/R-devel/library/base/html/libPaths.html
so a simple `library(SparkR)` later in the .Rmd will work